src/Membership/MembershipRequest.php

```diff
@@ -76,26 +78,27 @@ class MembershipRequest implements MembershipInterface
     public $comEmail = false;
 
     /**
-     * @Assert\NotBlank(message="common.recaptcha.invalid_message", groups="Registration")
+     * @Assert\NotBlank(message="common.recaptcha.invalid_message", groups={"Registration"})
      * @AssertRecaptcha(groups={"Registration"})
      */
     public $recaptcha;
```

Comment:
I've done a better handling of the recaptcha on `auth` repo, I'll import it as well to get rid of all these public properties in another PR.

Reply:
Yeap !